manifests/server.pp

```diff
@@ -478,7 +478,7 @@
   String $pam_module_arguments              = 'login',
   Boolean $management                       = false,
   String $management_ip                     = 'localhost',
-  Integer $management_port                  = 7505,
+  Variant[Integer,String] $management_port  = 7505,
```

Comment:
Is there any string valid, or only 'unix'? Otherwise we could do something like `Variant[Integer,Enum['unix']]`.

---

Could you also enhance this by removing the `Integer` type and replace it with `Stdlib::Port::Unprivileged` from https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port/unprivileged.pp ? This requires a bump of the minimal stdlib version to 4.25.0 in the metadata.json:
https://github.com/voxpupuli/puppet-openvpn/blob/4814cd7529f707f675ef4adff4ff76431303a938/metadata.json#L56